bitmap.h

```diff
@@ -47,11 +47,12 @@ static inline uint32_t get_free_blocks(struct super_block *sb, uint32_t len)
 {
     struct simplefs_sb_info *sbi = SIMPLEFS_SB(sb);
     uint32_t ret = get_first_free_bits(sbi->bfree_bitmap, sbi->nr_blocks, len);
+    uint32_t i;
     if (!ret) /* No enough free blocks */
         return 0;
 
     sbi->nr_free_blocks -= len;
-    for (uint32_t i = 0; i < len; i++) {
+    for (i = 0; i < len; i++) {
```

Comment:
Place the declaration of `uint32_t i` here.

Reply:
If declared here, it will cause the error: ‘for’ loop initial declarations are only allowed in C99 or C11 mode in ubuntu 22.04.

Reply:
> If declared here, it will cause the error: ‘for’ loop initial declarations are only allowed in C99 or C11 mode in ubuntu 22.04.

You are right. At past, I specified `-std=gnu99`, but I eliminated the flag for the sake of compatibility.